edx_proctoring/tests/test_views.py

```diff
@@ -855,182 +855,6 @@ def test_pagination_maintains_query_params(self):
         self.assertIn(text_search_string, next_url)
         self.assertIn(statuses_filter_string, next_url)
 
-    def test_partial_text_search_username(self):
```

Comment:
Why were all these test cases removed?

Reply:
edx_proctoring is no longer doing the username and email searching - it has been moved to the LMS as part of the `get_enrollments_can_take_proctored_exams ` method. I wrote tests in the LMS to capture this functionality.

Reply:
Got it. But I feel like these tests also cover the restful API interface on the endpoint here. Therefore, keeping them here won't hurt. We should have double coverage here.

Reply:
We're mocking out the enrollments service, so I would need to eitherreplicate the user searching there, which I don't think we want to do, or I'd need to import the function from the LMS here. Are you suggesting the second?

Reply:
I see. So, if we keep mocking here, the tests are no longer relevant. 
That said, I believe you can update one of these tests to check the mock is called with text search passed into the mock. That way, you leave maybe 1 test to ensure the text search interface is implemented. I am not suggesting you need to import from LMS